Esri/arcgis-maps-sdk-kotlin-samples
# Generate offline map using android jetpack's workmanager

## Description
PR to add a new Kotlin sample _"Generate offline map using workmanager"_ in `Edit and Manage Data` category.

## Links and Data
Sample Epic: `runtime/kotlin/issues/1731`

## What To Review
-  Review the code to make sure it is easy to follow like other samples on Android
- `README.md` and `README.metadata.json` files

## How to Test
Run the sample on the sample viewer or the repo.
Please check the README for more information - [README.md](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/tree/kaushikrw/generate-offline-map-workmanager/generate-offline-map-using-workmanager)

## Notes
If the checker fails on the README stage at `raise Exception(f'Wrong letter case for word: "{word}" in title.')` its because the checker expects only the first word of the title to be upper case. Please ignore it as we want to preserve the title casing for `Android Jetpack's WorkManager`
